README.md

```diff
@@ -82,6 +92,30 @@ Dict{String,Any} with 6 entries:
   "content_length"     => 193660
   "state"              => "completed"
 ```
+# Multiple credentials
+In case you want to use multiple api-tokens for different requests, you can specify the token to use with each different request.
+To do so you can have multiple versions of a `.cdsapirc` file stored somewhere. These files must be in the same format as the classic `.cdsapirc` file.
+
+To use them, parse them and pass the result to the scoped value `CDSAPI.auth`:
+```julia
+using CDSAPI
+
+dataset = "reanalysis-era5-single-levels"
+request = """ #= some request =# """
+
+_url1, _key1 = CDSAPI.credentialsfromfile("path/to/credential/file1")
+_url2, _key2 = CDSAPI.credentialsfromfile("path/to/credential/file2")
```

Comment:
Please adjust this example to avoid the `credentialsfromfile` internal function. Sorry for insisting on this point, but we need to be super picky with *public APIs*.